# Index Error:in Python occurs when you try to access an index in a sequence 
# (like a list or tuple) that is outside the valid range of indices for that sequence.
# The error typically indicates that
# the index you're trying to access does not exist in the sequence.

my_list = [1,2,3,5,6,7]
print(my_list[1])

# print(my_list[10])
